selfdrive/controls/lib/lead_mpc_lib/generator.cpp

```diff
@@ -34,8 +33,8 @@ int main( )
   Function h;
   h << exp(0.3 * NORM_RW_ERROR(v_ego, v_l, d_l)) - 1;
   h << (d_l - desired) / (0.05 * v_ego + 0.5);
-  h << a_ego * (0.1 * v_ego + 1.0);
-  h << j_ego * (0.1 * v_ego + 1.0);
+  h << a_ego;
```

Comment:
Why remove this speed dependent cost? Acceleration is more uncomfortable at higher speeds.

Reply:
This isn't final yet, in it's current state, there is no cost on acceleration, only jerk